### PR TITLE
feat(FN-3449-3450): content changes to reconciliation for report page

### PR DESCRIPTION
**Pull request by Zainzzkk**
## Introduction :pencil2:
This PR makes content changes to the utilisation report reconciliation for report page

## Resolution :heavy_check_mark:
* Content changes on nunjucks template
* Updated test

### CODE REVIEW

File: trade-finance-manager-ui/component-tests/utilisation-reports/utilisation-report-reconciliation-for-report.component-test.ts [hunk 1]
```diff
@@ -70,11 +70,11 @@ describe(page, () => {
     wrapper.expectText(`${premiumPaymentsTabSelector} h2[data-cy="premium-payments-heading"]`).toRead('Premium payments');
     wrapper
       .expectText(`${premiumPaymentsTabSelector} p`)
-      .toMatch(/Enter received payments against reported fees by selecting them and then selecting the 'Add a payment' button./);
+      .toMatch(/Received payments are entered against reported fees through selection and then selection of the 'Add a payment' button./);
```
Comment (ttbarnes) [inline trade-finance-manager-ui/component-tests/utilisation-reports/utilisation-report-reconciliation-for-report.component-test.ts:73]: Strange that we're not using page selectors or `cy.assertText` here 😞 

Appreciate this PR is purely content changes though.

**Review by ttbarnes — APPROVED** [review]

File: trade-finance-manager-ui/component-tests/utilisation-reports/utilisation-report-reconciliation-for-report.component-test.ts (hunk 1 above)
Comment (Zainzzkk) [reply trade-finance-manager-ui/component-tests/utilisation-reports/utilisation-report-reconciliation-for-report.component-test.ts:73]: I would like to add e2e tests for this section later for the page content

File: trade-finance-manager-ui/templates/utilisation-reports/utilisation-report-reconciliation-for-report.njk [hunk 2]
```diff
@@ -121,8 +121,6 @@
 {% set keyingSheetHtml %}
   <h2 class="govuk-heading-l" data-cy="keying-sheet-heading">Keying sheet</h2>
 
-  <p class="govuk-body">Select payments and mark as done when the adjustments have been keyed into ACBS.</p>
```
Comment (ChristianMcc) [inline trade-finance-manager-ui/templates/utilisation-reports/utilisation-report-reconciliation-for-report.njk:124]: I think we only wanted to delete this for PDC_READ users not just everyone, so we want to wrap this in an:
`{% if userCanEdit %}`

Could you open a new PR to add this please?

Comment (Zainzzkk) [reply trade-finance-manager-ui/templates/utilisation-reports/utilisation-report-reconciliation-for-report.njk:124]: will readd in PR for 3451

**Pull request merged**